google/cloud/aiplatform/training_utils/cloud_profiler/__init__.py

```diff
@@ -15,13 +15,7 @@
 # limitations under the License.
 #
 
-try:
```

Comment:
It seems like removing this may throw if werkzeug is not installed but it won't raise the same informative message.

Also as a follow up, should werkzeung be added to setup.py?

Reply:
Added werkzeug within the try/except and updated setup.py